docs/self-managed/reference-architecture/reference-architecture.md

```diff
@@ -0,0 +1,149 @@
+
+## Architecture
+
+### Orchestration Cluster
+
+![Orchestration Cluster](./img/orchestration-cluster.jpg)
+
+### Web Modeler and Console
+
+![Web Modeler and Console](./img/management-cluster.jpg)
+
+### Orchestration Cluster vs Web Modeler and Console
+
+When designing a reference architecture, it's essential to understand the differences between an orchestration cluster and Web Modeler and Console self-managed. These components play crucial roles in the deployment and operation of processes, but they serve different purposes and include distinct components.
+
+#### Orchestration Cluster
+
+We refer to the orchestration or automation cluster to the core of Camunda.
+
+The included components are:
+
+- [Zeebe](/components/zeebe/zeebe-overview.md): A workflow engine for orchestrating microservices and managing stateful, long-running business processes.
+- [Operate](/components/operate/operate-introduction.md): A monitoring tool for visualizing and troubleshooting workflows running in Zeebe.
+- [Tasklist](/components/tasklist/introduction-to-tasklist.md): A user interface for managing and completing human tasks within workflows.
+- [Optimize]($optimize$/components/what-is-optimize/): An analytics tool for generating reports and insights based on workflow data.
+- [Identity](/self-managed/identity/what-is-identity.md): A service for managing user authentication and authorization.
+- [Connectors](/components/connectors/introduction.md): Pre-built integrations for connecting Zeebe with external systems and services.
+
+The orchestration cluster is central to Camunda's workflow automation, which focuses on managing and executing processes through interconnected components. Each component within the orchestration cluster is part of an integrated system that works together to provide end-to-end process orchestration. Zeebe handles workflow execution, while Operate provides monitoring, and other components such as Tasklist and Optimize add functionality like human task management and analytics. These components form a unified cluster that is tightly integrated to ensure seamless communication and data flow. This design ensures that all components are in sync, working collectively to maintain consistent state management, data integrity, and smooth process orchestration across the entire cluster. This architecture ensures reliable process execution with clear boundaries between each workflow engine's operation.
+
+#### Web Modeler and Console
+
+The Web Modeler and Console are designed to oversee and manage multiple orchestration clusters. It offers tools and interfaces for administrators and developers to monitor clusters and create BPMN models. The management cluster operates independently of the orchestration cluster and can function without requiring an orchestration cluster.
```

Comment:
I am not sure if this description applies anymore - possibly from when it was meant to be a unified management cluster (no longer used)? Unsure if the below is 100% accurate, but is more of what I understand:

```suggestion
Web Modeler and Console are designed to interact with multiple orchestration clusters. Console offers tools and interfaces for administrators to monitor clusters, and Web Modeler allows developers to create and deploy BPMN models. 

These components operate independently of both the orchestration cluster and each other, and do not require an orchestration cluster to function.
```

Reply:
true, I think it isn't used word wise anymore. Sadly